docs/components/RelatedReadList.js

```diff
@@ -1,16 +1,23 @@
 import React from "react";
 import clsx from "clsx";
 import Link from "@docusaurus/Link";
+import {v4 as uuidv4} from "uuid";
 
-export default function RelatedRead({readliststring}) {
-  const items = readliststring.split("|");
+export default function RelatedReadList({readlist}) {
   let readingList = [];
-  for (const item of items) {
-    const parts = item.split("?");
-    const tagStuff = tagInfo(parts[2]);
+  for (const item of readlist) {
+    // validate 
+    const text = item[0];
+    if (text.startsWith('/') || text.startsWith('#')) throw new Error(`readlist item "${text}" looks like a URL, did you forget the sequence?`)
+    const goTo = item[1];
+    if (!goTo.startsWith('/') || !goTo.startsWith('#')) throw new Error(`goTo parameter ${goTo} doesnt start with / or #: was there a mistake?`)
+    const tagStuff = tagInfo(item[2]);
+    if (tagStuff instanceof Error) throw tagStuff;
```

Comment:
added some validation... see if you agree with it. havent run the code yet

Reply:
Ya, the component relies on the array argument to have certain items in the proper order.
Not sure how much we validate that. But I understand your point.